packages/fiori-elements-writer/test/__snapshots__/alp.test.ts.snap

```diff
@@ -127,11 +129,13 @@ server:
     - name: sap-fe-mockserver
       beforeMiddleware: csp
       configuration:
-        service:
-          urlPath: /sap/opu/odata/sap/SEPMRA_ALP_SO_ANA_SRV
-          metadataXmlPath: ./webapp/localService/metadata.xml
-          mockdataRootPath: ./webapp/localService/data
-          generateMockData: true
+        mountPath: /
+        services:
+          - urlPath: /sap/opu/odata/sap/SEPMRA_ALP_SO_ANA_SRV
+            metadataXmlPath: ./webapp/localService/metadata.xml
```

Comment:
while you're at it you can rename metadataXmlPath to metadataPath and mockdataRootPath to mockdataPath :)

Reply:
Done with https://github.com/SAP/open-ux-tools/pull/533/commits/2f0d1de73bc031a6adf2df712175de8153fe5766